src/main/resources/application.yaml

```diff
@@ -12,6 +12,17 @@ no.nav.security.jwt:
     discoveryurl: ${AZURE_APP_WELL_KNOWN_URL}
     accepted_audience: ${AZURE_APP_CLIENT_ID}
     cookie_name: azure_token
+  client:
+    registration:
+      familie-klage-onbehalfof:
+        resource-url: ${FAMILIE_KLAGE_URL}
+        token-endpoint-url: ${AZUREAD_TOKEN_ENDPOINT_URL}
+        grant-type: urn:ietf:params:oauth:grant-type:jwt-bearer
+        scope: ${FAMILIE_KLAGE_SCOPE}
+        authentication:
+          client-id: ${AZURE_APP_CLIENT_ID}
+          client-secret: ${AZURE_APP_CLIENT_SECRET}
+          client-auth-method: client_secret_basic
```

Comment:
Dette er ikke sånn vi har gjort det for de andre applikasjonene. Der blir det definert i de andre configfilene. Siden vi bruker variabler vil dette bli likt alle steder, synes derfor det er mer naturlig å ha det her. Tanker?